.github/workflows/build.yaml

```diff
@@ -0,0 +1,72 @@
+name: build
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        uses: actions/checkout@master
+        with:
+            fetch-depth: 1
+      -
+        name: Set Username/Repo as ENV vars
+        run: echo "USER_REPO"=$(echo "$GITHUB_REPOSITORY" | awk '{print tolower($1)}' | sed -e "s/:refs//") >> $GITHUB_ENV
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - 
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - 
+        name: Local docker build (non-root image)
```

Comment:
Do this in a follow-up PR if you want, but we need to do a multi-arch build upon every commit, not just upon a release tag. This found issues in the past.

Reply:
I'll try to put up a PR by tonight to address this 👆

Reply:
@alexellis I've resolved it in a PR I've just put up
https://github.com/openfaas/faas-cli/pull/843